arbitrum-docs/build-decentralized-apps/how-to-use-timeboost.mdx

```diff
@@ -0,0 +1,425 @@
+  body: JSON.stringify({
+    jsonrpc: '2.0',
+    id: 'express-lane-tx',
+    method: 'timeboost_sendExpressLaneTransaction',
+    params: [
+      {
+        chainId: hexChainId,
+        round: `0x${currentRound.toString(16)}`,
+        auctionContractAddress: auctionContractAddress,
+        sequence: `0x${sequenceNumber.toString(16)}`,
+        transaction: serializedTransaction,
+        options: {},
+        signature: signature,
+      },
+    ],
+  }),
+});
+```
+
+The signature that needs to be sent is an Ethereum signature over the bytes encoding of the following information:
+
+- Hash of `keccak256("TIMEBOOST_BID")`
+- Chain id in hexadecimal, padded to 32 bytes
+- Auction contract address
+- Round number in hexadecimal, padded to 8 bytes
+- Sequence number in hexadecimal, padded to 8 bytes
+- Serialized transaction
+
+Here's an example to produce that signature:
+
+```tsx
+const hexChainId: `0x${string}` = `0x${Number(publicClient.chain.id).toString(16)}`;
+
+const transaction = await walletClient.prepareTransactionRequest(...);
+const serializedTransaction = await walletClient.signTransaction(transaction);
+
+const signatureData = concat([
+  keccak256(toHex('TIMEBOOST_BID')),
+  pad(hexChainId),
+  auctionContract,
+  toHex(numberToBytes(currentRound, { size: 8 })),
+  toHex(numberToBytes(sequenceNumber, { size: 8 })),
+  serializedTransaction,
+]);
+const signature = await account.signMessage({
+  message: { raw: signatureData },
+});
+```
+
+When sending the request, the sequencer will return an empty result with an HTTP status `200` if it received the request correctly. If the result returned contains an error message, it'll mean that something went wrong. Following are some of the error messages that can help us understand what's happening:
+
+| Error                         | Description                                                           |
+| ----------------------------- | --------------------------------------------------------------------- |
+| `MALFORMED_DATA`              | wrong input data, failed to deserialize, missing certain fields, etc. |
+| `WRONG_CHAIN_ID`              | wrong chain id for the target chain                                   |
+| `WRONG_SIGNATURE`             | signature failed to verify                                            |
+| `BAD_ROUND_NUMBER`            | incorrect round, such as one from the past                            |
+| `NOT_EXPRESS_LANE_CONTROLLER` | the sender is not the express lane controller                         |
+| `NO_ONCHAIN_CONTROLLER`       | there is no defined, on-chain express lane controller for the round   |
+
+::::info What happens if you're not the express lane controller?
+
+If you are not the express lane controller and you try to submit a transaction to the express lane, the sequencer will respond with the error `NOT_EXPRESS_LANE_CONTROLLER` or `NO_ONCHAIN_CONTROLLER`.
+
+::::
+
+## How to transfer the right to use the express lane to someone else
```

Comment:
Should this be Step #5 ?

Reply:
Same as above, these are different flows. Using "Steps" leads to think that they are all part of one process.